packages/components/src/theme-type-badge/bundled-badge/style.scss

```diff
@@ -1,4 +1,4 @@
-.woocommerce-bundled-badge {
+.bundled-badge {
 	--woocommerce-purple-50: #7f54b3;
```

Comment:
this is woo specific so will need changing at some point 🤔

Reply:
Same thing applies to line 13 below

Reply:
Good catch! It's supposed to be generic here now: https://github.com/Automattic/wp-calypso/pull/84483/commits/078d3465c224b5fc5cd402205c77a2a422a112f8